src/Pages/MainPage/MainBody/Best.js

```diff
@@ -1,32 +1,63 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
 import './Best.scss';
 
-export default class Best extends Component {
+class Best extends Component {
+  constructor() {
+    super();
+    this.state = { bestList: [] };
+  }
+  componentDidMount() {
+    // fetch('http://localhost:8000/products/main-best')
+    fetch('/data/MainData/Best.json')
```

Comment:
```suggestion
  }
  
  componentDidMount() {
    // fetch('http://localhost:8000/products/main-best')
    fetch('/data/MainData/Best.json')
```
구별될 수 있는 요소들은 한줄 씩 다 띄워주시면 좋겠습니다!

Reply:
넵!